src/core/Header/HeaderBreadcrumbs.tsx

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+
+export type HeaderBreadcrumbsProps = {
+  items: React.ReactNode[];
+};
+
+/**
+ * Header breadcrumbs, adds chevron between each provided items.
```

Comment:
Either here or in `HeaderButton`, I would add something about how the behavior changes in slim mode

Reply:
Updated `HeaderButton`, breadcrumbs does not really do anything specific with slim.